src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
```

Comment:
2. Error Handling: The error message "This is a deliberate error for PR review" is not a helpful error message for end-users. It's important to provide meaningful error messages to help users understand and resolve issues when things go wrong.